### PR TITLE
Convert /enterprise to Protocol (#7352)

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- Remove old assets
- Add new ones
- Re-code to not use table anymore
- Update look of page

## Issue / Bugzilla link

See #7352

## Testing

https://bedrock-demo-shobson.oregon-b.moz.works/en-US/firefox/enterprise/

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Note: the /signup and /signup/thanks pages will be addressed in a separate PR once #7141 is merged.

File: media/css/firefox/enterprise.scss [hunk 1]
```diff
@@ -2,297 +2,157 @@
...
-
-            .logo {
-                margin: $enterprise-grid-spacing 0 40px;
+// Hero
+
+.mzp-c-hero.t-enterprise {
+    // dark colour fallback incase svg image doesn't load/display
+   background: $color-off-black;
+   background: url('/media/img/firefox/enterprise/ent-noodles.svg') top left no-repeat,
+            linear-gradient(90deg, #fff 1000px, $color-off-black 1000px, $color-off-black 2425px, #fff 2425px, #fff 3425px) top center repeat;
```
Comment (alexgibson) [inline media/css/firefox/enterprise.scss]: Nit: indentation

File: media/css/firefox/enterprise.scss [hunk 2]
```diff
@@ -2,297 +2,157 @@
...
+       margin-bottom: 100px;
+   }
+
+   .mzp-c-hero-image {
+        display: none;
+    }
+
+    @media #{$mq-sm} {
+        margin-bottom: $layout-md;
+        min-height: 560px; // too keep bg image from being cut off
```
Comment (alexgibson) [inline media/css/firefox/enterprise.scss]: Nit: typo `too keep`

**Comment by alexgibson** [comment]
Not a blocker as the page isn't yet localized, but RTL styles for the hero need a little work:

image

I also see a thin border on the bottom of the hero image at small screen sizes:

image

**Comment by alexgibson** [comment]
Hero backgroundlooks a littke funky in IE 10 & 11

Screenshot 2019-06-28 at 14 30 29

It also has a border in Edge:

Screenshot 2019-06-28 at 14 32 18

**Comment by stephaniehobson** [comment]
I tweaked my linear gradient background fallbacks and I think I got rid of the thin lines around the image.

WFT is IE smoking? 😡 (still working on this)

**Comment by stephaniehobson** [comment]
Ready for re-review 😅

**Comment by stephaniehobson** [comment]
RTL styles are super basic for now since the page is not planned to be localized.

File: bedrock/firefox/templates/firefox/enterprise/index.html [hunk 3]
```diff
@@ -23,75 +25,64 @@
...
-  </section>
+  {% call hero(
+    title=_('Firefox Quantum for Enterprise'),
+    desc=_('Speed up your business by giving employees a managed version of the super-fast new Firefox. <em>Now with Windows Group Policy support.</em>'),
+    class='mzp-has-image mzp-t-dark mzp-t-product-firefox t-enterprise',
+    include_cta=True,
+    image_url='firefox/enterprise/fx-browser-img.svg',
+    heading_level=1
+  ) %}
+  <a href="{{ url('firefox.enterprise.signup') }}" class="mzp-c-button mzp-t-product">{{ _('Find Out More') }}</a>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/enterprise/index.html]: This link has a functional test associated with it that relies upon the CSS class `qa-sign-up`. We should retain that class here in order for the test run to pass.

File: media/css/firefox/enterprise.scss [hunk 4]
```diff
@@ -2,297 +2,169 @@
...
-                        auto,
-                        auto,
-                        100%;
+// Hero
+
+.mzp-c-hero.t-enterprise {
+    // dark colour fallback incase svg image doesn't load/display
+    background: $color-off-black;
+    background-image: url('/media/img/firefox/enterprise/ent-noodles.svg'),
+        linear-gradient(transparent 550px, #fff 550px, #fff 1000px);
```
Comment (alexgibson) [inline media/css/firefox/enterprise.scss]: Nit: indent inline with `url`.

File: media/css/firefox/enterprise.scss [hunk 5]
```diff
@@ -2,297 +2,169 @@
...
+    @media #{$mq-sm} {
+        background-position: top left, top center;
+        margin-bottom: $layout-md;
+        min-height: 680px - $layout-xl; // to keep bg image from being cut off
+        padding-top: $layout-xl;
+    }
+
+    @media #{$mq-md} {
+        background-image: url('/media/img/firefox/enterprise/ent-noodles-wide.svg'),
+                    linear-gradient(transparent 680px, #fff 680px, #fff 1200px);
```
Comment (alexgibson) [inline media/css/firefox/enterprise.scss]: Nit: indent inline with `url()`

**Comment by alexgibson** [comment]
Oh, you're on PTO! This is so close to finished I opened a new PR that fixes the test, so this doesn't need to sit until you're back :)

Closing in favor of https://github.com/mozilla/bedrock/pull/7362

**Pull request closed without merging**